### PR TITLE
chore(dependabot): add dependabot

### PR DESCRIPTION
**Pull request by sad270**
| Q             | A
| ------------- | ---
| Branch?       | 3.1
| Tickets       | 
| License       | MIT
| Doc PR        | 

Add dependabot to update github actions.

Configuration is comming from https://github.com/api-platform/api-platform/blob/main/.github/dependabot.yml

### CODE REVIEW

**Comment by soyuka** [comment]
we're using our own `update-js` script there is no package-lock on this project I think

**Comment by sad270** [comment]
Hello, i'm maybe wrong but `update-js` will update js js packages, in this PR dependabot will update github action `uses`.

**Comment by sad270** [comment]
Hello, i'm maybe wrong but `update-js` will update js js packages, in this PR dependabot will update github action `uses`.

**Comment by sad270** [comment]
In https://github.com/api-platform/core/pull/5946 I did it manually 😅 but if it is automatically updated it will be better ! With dependabot or with another tool.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+version: 2
+
+updates:
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: monthly
+
```
Comment (dunglas) [inline .github/dependabot.yml:9]:
```suggestion
```

**Review by dunglas — APPROVED** [review]

**Comment by dunglas** [comment]
Y the way, as we keep the `yarn.lock` file, we may also replace `update-js.js` by Dependabot (in a next PR).

File: .github/dependabot.yml (hunk 1 above)
Comment (sad270) [reply .github/dependabot.yml]: 👍🏻 it's removed !

**Comment by dunglas** [comment]
One last thing: this should target the 3.1 branch

**Comment by sad270** [comment]
> One last thing: this should target the 3.1 branch

@dunglas target changed it's `3.1` now 😄

**Comment by dunglas** [comment]
Thank you!!

**Pull request merged**